recipes/mosquitto/conda-forge.yml

```
@@ -0,0 +1,4 @@
+build_platform:
+  linux_aarch64: linux_64
```

If you're including aarch64 is there a reason to not do ppc64le as well?
```suggestion
  linux_aarch64: linux_64
  linux_ppc64le: linux_64
```

---

not really, will add!